ckan/lib/helpers.py

```diff
@@ -1020,7 +1020,8 @@ def build_nav_main(*args):
 def build_nav_icon(menu_item, title, **kw):
     '''Build a navigation item used for example in ``user/read_base.html``.
 
-    Outputs ``<li><a href="..."><i class="icon.."></i> title</a></li>``.
+    Outputs ``<li class="nav-item nav-link"><a href="..."><i class="icon..">
```

Comment:
Inline code operator used for multi-line code output. I think this is one of the test failures.